Alpaca.Markets/Interfaces/IHistoricalQuoteV2.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Alpaca.Markets
+{
+    /// <summary>
+    /// Encapsulates historical trade information from Polygon REST API.
+    /// </summary>
+    public interface IHistoricalQuoteV2
+    {
+        /// <summary>
+        /// Gets SIP timestamp
+        /// </summary>
+        Int64 SipTimestamp { get; }
```

Comment:
It's OK to expose this timestamp as integer value because it can be used for pagination in request calls but in general, it's better to provide `DateTime` values instead of raw integers.